service/src/main/java/com/epam/esm/service/service/certificate/CertificateServiceImpl.java

```diff
@@ -174,12 +209,22 @@ private void parsePaginationParameter(CertificateFinder finder,
         }
     }
 
-    private void parseSortParameter(CertificateFinder finder, String parameterName, List<String> parameterValues) {
-        finder.sortBy(CertificateSortingParameters.getEntryByParameter(parameterName).getValue(),
-                SortDirection.parseAscDesc(parameterValues.get(0)));
+    private void addSortingToFinder(CertificateFinder finder,
+                                    String sortingParameter,
+                                    SortDirection sortDirection) {
+        finder.sortBy(sortingParameter, sortDirection);
```

Comment:
please review if you really need all these one line methods

Reply:
It is made for refactoring and simplifying more complicated methods.